src/MFRC522.cpp

```diff
@@ -481,18 +487,22 @@ MFRC522::StatusCode MFRC522::PCD_CommunicateWithPICC(	byte command,		///< The co
 	// In PCD_Init() we set the TAuto flag in TModeReg. This means the timer automatically starts when the PCD stops transmitting.
 	// Each iteration of the do-while-loop takes 17.86μs.
```

Comment:
update doc please

Reply:
Would you prefer to have `PCD_CommunicateWithPICC` receive a `timeout` parameter with a default of 36 for backwards compatibility?